### PR TITLE
Add KDE@ report

### PR DESCRIPTION
**Pull request by adriaandegroot**

### CODE REVIEW

**Comment by adriaandegroot** [comment]
I expect I've made serious ASCIIDoc errors here, but felt it was more important to throw the content over the wall than to fiddle with layouts.

File: 2021q2/kde.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+=== KDE on FreeBSD
+
+Links: +
+link:https://freebsd.kde.org/[KDE FreeBSD] URL: link:https://freebsd.kde.org/[https://freebsd.kde.org/] +
+link:https://community.kde.org/FreeBSD[KDE Community FreeBSD] URL: link:https://community.kde.org/FreeBSD[https://community.kde.org/FreeBSD]
+
+Contact: Adriaan de Groot <kde@FreeBSD.org>
+
+The KDE on FreeBSD project aims to package all of the software produced by the KDE Community for the FreeBSD ports tree.
+The software includes a full desktop environment called KDE Plasma, graphics applications, instant-messengers, a video-editing suite, as well as a tea timer and hundreds of other applications that can be used on any FreeBSD machine.
```
Comment (pauamma) [inline 2021q2/kde.adoc:10]: I'd make that "instant messengers" (without the hyphen).

File: 2021q2/kde.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  and has not landed in ports; it will probably wait until after the next
+  quarterly is cut.
+
+## Highlights
+
+- Poppler, CMake, polkit .. these underpinnings of the Free Desktop on
+  FreeBSD all received their regular updates. 
+- *textproc/kdiff3* had some serious issues with 3-way-merges. This was
+  reported by Ed Maste, and turned out to have fixes upstream.
+- *x11/konsole* the terminal emulator would not accept `/bin/sh` as a
```
Comment (pauamma) [inline 2021q2/kde.adoc:53]: I'd add commas: "- *x11/konsole*, the terminal emulator, would not accept ..."

File: 2021q2/kde.adoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  shell; this is more of an issue on FreeBSD than on Linux where `bash`
+  is a very common choice. This was resolved upstream and landed in ports.
+- The regular KDE Frameworks releases (monthly) and KDE Plasma releases
+  (monthly) and KDE Gear all were announced upstream and landed in FreeBSD
+  ports without a fuss. Upstream CI continues to be updated with the latest
+  FreeBSD ports so that CI reflects well what the KDE-on-FreeBSD team
+  delives to FreeBSD users.
```
Comment (pauamma) [inline 2021q2/kde.adoc:60]: s/delives/delivers/

**Pull request closed without merging**